public/locales/en.json

```diff
@@ -126,6 +127,9 @@
     },
     "subdomainFavouriteTitle": "Favourite Subdomains"
   },
+  "expiry": {
+    "reminder": "Set Renewal Reminder"
+  },
```

Comment:
Please duplicate the same in `public/locals/en.json`.

You can just run `yarn i18n` to generate it